source/dom.js

```diff
@@ -79,11 +81,11 @@ var dom = module.exports = {
      * Find all nodes by selector
      *
      * @param {String} selector
-     * @param {Node} node
+     * @param {Node} [node]
      * @returns {NodeList}
```

Comment:
(It helps because I’ll see there’s a review pending and won’t forget to fix the issues.)